src/sentry/api/endpoints/organization_user_teams.py

```diff
@@ -26,6 +27,8 @@ def get(self, request, organization):
             queryset = Team.objects.filter(
```

Comment:
Note that the superuser path might be really slow on large orgs, since it'll essentially return every team, and every project that belongs to the teams (so projects will be duplicated)

Reply:
gotcha, yea it seems like for super user decomposing the org details won't do much since all this data was already being fetched, but hopefully regular customers that are only part of a few teams will be able to reap larger performance benefits